src/VTMap.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Map, TileLayer, Polygon, Marker, Polyline } from "react-leaflet";
+import borderData from "./Components/border.js";
+
+
+class VTMap extends React.Component {
+  getCorners = (geoJson) => {
+    geoJson.geometry.coordinates[0].map((coordArray) => {
+      return coordArray;
+    });
+  };
+
+  render() {
+    this.getCorners(borderData);
+
+    let vtBorder = borderData.geometry.coordinates[0].map((coordSet) => {
+      return [coordSet[1], coordSet[0]];
+    });
+
+    return (
+      <Map
+        center={this.props.coords}
+        style={{ height: "500px", width: "500px" }}
+        zoom={this.props.zoom}
+        onClick={this.addMarker}
+        dragging={false} zoomControl={false} scrollWheelZoom={false} touchZoom={false} doubleClickZoom={false}
```

Comment:
You missed the zoom controlls using 'shift -' and 'shift +'. I dont remember the variable name that they give to that one though